### PR TITLE
[Enhancement] correct error message when creating mv in external catalog

### PR DESCRIPTION
**Pull request by murphyatwork**
Why I'm doing:
- Creating mv in external catalog would report a wired exception, making users confused

What I'm doing:
- Correct the error message when creating mv

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Review by chaoyli — DISMISSED** [review]

**Review by Youngwb — DISMISSED** [review]

File: fe/fe-core/src/main/java/com/starrocks/catalog/InternalCatalog.java [hunk 1]
```diff
@@ -25,4 +27,8 @@ public InternalCatalog(long id, String name, Map<String, String> config, String
         super(id, DEFAULT_INTERNAL_CATALOG_NAME, config, comment);
     }
 
+    public static boolean isDefault(TableName name) {
```
Comment (ABingHuang) [inline fe/fe-core/src/main/java/com/starrocks/catalog/InternalCatalog.java:30]:
```suggestion
    public static boolean isFromDefault(TableName name) {
```
?

File: fe/fe-core/src/main/java/com/starrocks/catalog/InternalCatalog.java [hunk 2]
```diff
@@ -25,4 +27,8 @@ public InternalCatalog(long id, String name, Map<String, String> config, String
...
+        return DEFAULT_INTERNAL_CATALOG_NAME.equalsIgnoreCase(name.getCatalog());
```
Comment (ABingHuang) [inline fe/fe-core/src/main/java/com/starrocks/catalog/InternalCatalog.java:31]: name != null && DEFAULT_INTERNAL_CATALOG_NAME.equalsIgnoreCase(name.getCatalog());
?

File: fe/fe-core/src/main/java/com/starrocks/catalog/InternalCatalog.java (hunk 1 above)
Comment (mofeiatwork) [reply fe/fe-core/src/main/java/com/starrocks/catalog/InternalCatalog.java]: fixed

File: fe/fe-core/src/main/java/com/starrocks/catalog/InternalCatalog.java (hunk 2 above)
Comment (mofeiatwork) [reply fe/fe-core/src/main/java/com/starrocks/catalog/InternalCatalog.java]: fixed

**Review by Youngwb — APPROVED** [review]

**Review by packy92 — APPROVED** [review]

**Pull request merged**